lib/twitter_cldr/js/compiler.rb

```diff
@@ -77,62 +86,81 @@ def compile_test(options = {})
-          :character_set                   => TwitterCldr::Js::Renderers::Parsers::CharacterSetRenderer,
-          :symbol_table                    => TwitterCldr::Js::Renderers::Parsers::SymbolTableRenderer,
-          :parser                          => TwitterCldr::Js::Renderers::Parsers::ParserRenderer,
-          :segmentation_parser             => TwitterCldr::Js::Renderers::Parsers::SegmentationParserRenderer,
-          :unicode_regex_parser            => TwitterCldr::Js::Renderers::Parsers::UnicodeRegexParserRenderer,
-          :token                           => TwitterCldr::Js::Renderers::Tokenizers::TokenRenderer,
-          :composite_token                 => TwitterCldr::Js::Renderers::Tokenizers::CompositeTokenRenderer,
-          :tokenizer                       => TwitterCldr::Js::Renderers::Tokenizers::TokenizerRenderer,
-          :segmentation_tokenizer          => TwitterCldr::Js::Renderers::Tokenizers::SegmentationTokenizerRenderer,
-          :unicode_regex_tokenizer         => TwitterCldr::Js::Renderers::Tokenizers::UnicodeRegexTokenizerRenderer,
-          :rbnf_tokenizer                  => TwitterCldr::Js::Renderers::Tokenizers::RBNFTokenizerRenderer,
-          :number_tokenizer                => TwitterCldr::Js::Renderers::Tokenizers::NumberTokenizerRenderer,
-          :pattern_tokenizer               => TwitterCldr::Js::Renderers::Tokenizers::PatternTokenizerRenderer,
-          :numbers                         => TwitterCldr::Js::Renderers::Numbers::NumbersRenderer,
-          :rbnf                            => TwitterCldr::Js::Renderers::Numbers::RBNF::RBNFRenderer,
-          :number_data_reader              => TwitterCldr::Js::Renderers::Numbers::RBNF::NumberDataReaderRenderer,
-          :rbnf_formatters                 => TwitterCldr::Js::Renderers::Numbers::RBNF::FormattersRenderer,
-          :rbnf_rule                       => TwitterCldr::Js::Renderers::Numbers::RBNF::RuleRenderer,
-          :rbnf_rule_group                 => TwitterCldr::Js::Renderers::Numbers::RBNF::RuleGroupRenderer,
-          :rbnf_rule_set                   => TwitterCldr::Js::Renderers::Numbers::RBNF::RuleSetRenderer,
-          :rbnf_substitution               => TwitterCldr::Js::Renderers::Numbers::RBNF::SubstitutionRenderer,
-          :rbnf_rule_parser                => TwitterCldr::Js::Renderers::Numbers::RBNF::RuleParserRenderer,
-          :plural                          => TwitterCldr::Js::Renderers::Numbers::RBNF::PluralRenderer,
-          :range                           => TwitterCldr::Js::Renderers::Utils::RangeRenderer,
-          :range_set                       => TwitterCldr::Js::Renderers::Utils::RangeSetRenderer,
-          :code_points                     => TwitterCldr::Js::Renderers::Utils::CodePointsRenderer
+          :settings                        => TwitterCldr::Js::Renderers::ImplementationRenderers::Settings::SettingsRenderer,
```

Comment:
It's not like our namespacing wasn't scary before, but now full names are noticeably longer. How about `Renderers::Implementation` and `Renderers::Data`? Also, I'm not a big fan of cryptic abbreviation, but I feel like good old `Impl` could help here as well. I'm usually against that kind of things, but if you guys don't feel too bad about it, I think `Renderers::Impl` might be fine in our extreme case.